app/lib/actions/katello/organization/simple_content_access/toggle.rb

```diff
@@ -24,10 +25,23 @@ def plan(organization_id)
             plan_action(::Actions::Katello::Organization::ManifestRefresh, organization)
           end
 
+          def failure_notification(plan)
+            task_error_notification.deliver!(
+              :subject => subject_organization,
+              :task => get_foreman_task(plan)
+            )
+          end
+
+          def success_notification(_plan)
+            task_success_notification.deliver!(
+              subject_organization
+            )
+          end
+
           private
 
           def consumer
-            @consumer ||= ::Katello::Resources::Candlepin::UpstreamCandlepinResource.upstream_consumer
+            @consumer ||= @organization.owner_details['upstreamConsumer']
```

Comment:
had to change this to enable dealing with an org that is not `Organization.current`

Reply:
Nice. I really dislike the areas of our code that rely on Org.current